monkey/infection_monkey/credential_store/i_credentials_store.py

```diff
@@ -0,0 +1,19 @@
+import abc
+from typing import Mapping
+
+
+class ICredentialsStore(metaclass=abc.ABCMeta):
+    @abc.abstractmethod
+    def add_credentials(self, credentials_to_add: Mapping = {}) -> None:
+        """
+        Method that adds credentials to the CredentialStore
+        :param Credentials credentials: The credentials which will be added
+        """
+
+    @abc.abstractmethod
+    def get_credentials(self) -> Mapping:
+        """
+        Method that gets credentials from the ControlChannel
```

Comment:
Don't couple the interface to the `ControlChannel`. Credentials could be gotten from text files on the local disk, some external API, etc. The whole purpose of the interface is so that anything that uses `ICredentialsStore` doesn't have to worry about where credentials come from.